### PR TITLE
IPC Blood Trails are black for now.

### PR DESCRIPTION
**Pull request by Kapu1178**
## About The Pull Request
Will probably rewrite this in K-Blood

## Why It's Good For The Game
Immersion

## Changelog
:cl:
tweak: IPC blood trails are black now
/:cl:

### CODE REVIEW

**Comment by Tsar-Salat** [comment]
Do they have blood trails now?

wack.

**Comment by mcmeiler** [comment]
BasedBasedBasedBasedBasedBasedBasedBasedBasedBasedBasedBasedBasedBasedBasedBasedBasedBasedBasedBasedBasedBasedBasedBasedBasedBasedBasedBasedBasedBasedBasedBasedBasedBasedBasedBasedBasedBasedBasedBasedBasedBasedBased

**Comment by PowerfulBacon** [comment]
why do IPCs have blood trails?

**Comment by Kapu1178** [comment]
> 
> 
> why do IPCs have blood trails?

They have oil for blood now.

File: code/modules/mob/living/living.dm [hunk 1]
```diff
@@ -689,9 +689,13 @@
...
 						TH.transfer_mob_blood_dna(src)
 
-/mob/living/carbon/human/makeTrail(turf/T)
+						if(spec_color)
+							TH.color = spec_color
+
+/mob/living/carbon/human/makeTrail(turf/T, turf/start, direction, spec_color)
 	if((NOBLOOD in dna.species.species_traits) || !bleed_rate || bleedsuppress)
 		return
+	spec_color = dna?.species.blood_color
```
Comment (Archanial) [inline code/modules/mob/living/living.dm]: You can't get here if your mob doesn't have dna (see check above)

Comment (Archanial) [reply code/modules/mob/living/living.dm]:
```suggestion
	spec_color = dna.species.blood_color
```

Comment (Kapu1178) [reply code/modules/mob/living/living.dm]: make a suggestion and ill web edit it. im not booting up VSC to remove a question mark

**Review by Archanial — APPROVED** [review]

**Comment by francinum** [comment]
This is a good enough hack for now

**Pull request merged**